Simplenote/AuthViewController+Swift.swift

```diff
@@ -153,10 +146,35 @@ extension AuthViewController {
         refreshInterface(animated: true)
         ensureUsernameIsFirstResponder()
     }
+    
+    @IBAction
+    func performMainAction(_ sender: Any) {
+        performSelector(onMainThread: mode.primaryActionSelector, with: nil, waitUntilDone: false)
+    }
+    
+    @IBAction
+    func performSecondaryAction(_ sender: Any) {
+        guard let secondaryActionSelector = mode.secondaryActionSelector else {
+            return
+        }
+        
+        performSelector(onMainThread: secondaryActionSelector, with: nil, waitUntilDone: false)
+    }
```

Comment:
Same note as above: Primary and Secondary actions are determined by the `AuthenticationMode`, thus, simplifying our logic a bit